### PR TITLE
Created `config.json` files

### PR DESCRIPTION
**Pull request by xmwx38**
- `app.js` files edited to use `config.json`
- `config.json` files created

I based these edits on [GT2KING's YouTube tutorials](https://www.youtube.com/playlist?list=PLf7pCRJduyrCDjHLgG9oDHbYUF-WmLUSt).   
His tutorials were created with the [node-steam-guide](https://github.com/andrewda/node-steam-guide/) as reference.

### CODE REVIEW

File: Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js [hunk 1]
```diff
@@ -30,7 +31,7 @@ app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'hbs');
 
 app.use(session({
-    secret: 'some secret string',
+	secret: config.secretString, // some secret string
```
Comment (andrewda) [inline Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js]: Spacing is messed up here.

File: Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js [hunk 2]
```diff
@@ -19,7 +20,7 @@ passport.deserializeUser((obj, done) => {
 passport.use(new SteamStrategy({
 		returnURL: 'http://localhost:3037/auth/steam/return',
 		realm: 'http://localhost:3037/',
-		apiKey: 'your api key'
+		apiKey: config.apiKey	// Steam API key
```
Comment (andrewda) [inline Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js]: Should be fine without the comment.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ passport.deserializeUser((obj, done) => {
 passport.use(new SteamStrategy({
 		returnURL: 'http://localhost:3037/auth/steam/return',
 		realm: 'http://localhost:3037/',
-		apiKey: 'your Steam API key'
+		apiKey: config.apiKey	// Steam API key
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js]: Same as above – comment probably not needed.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js [hunk 4]
```diff
@@ -60,7 +60,7 @@ passport.use(new SteamStrategy({
 io.use(passportSocket.authorize({
 	cookieParser: cookieParser,
 	key: 'U_SESSION',
-	secret: 'some secret string',
+	secret: config.secretString,	// some secret string
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js]: Same as above – comment probably not needed.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js [hunk 5]
```diff
@@ -83,7 +83,7 @@ app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'hbs');
 
 app.use(session({
-    secret: 'some secret string',
+    secret: config.secretString,	// some secret string
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js]: Same as above – comment probably not needed.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/config.json [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{
+	"apiKey":"",
+	"secretString":""
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/config.json]: Spaces after the colon.

File: Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/config.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
+	"apiKey":"",
+	"secretString":""
```
Comment (andrewda) [inline Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/config.json]: Spaces after the colon.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/config.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{
+	"username":"",
+	"password":"",
+	"sharedSecret":"",
+	"apiKey":"",
+	"secretString":""
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/config.json]: Spaces after the colon.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js [hunk 9]
```diff
@@ -106,7 +107,7 @@ app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'hbs');
 
 app.use(session({
-    secret: 'some secret string',
+    secret: config.secretString,	// some secret string
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: Same as above – comment probably not needed.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js [hunk 10]
```diff
@@ -67,7 +68,7 @@ passport.use(new SteamStrategy({
 io.use(passportSocket.authorize({
 	cookieParser: cookieParser,
 	key: 'U_SESSION',
-	secret: 'some secret string',
+	secret: config.secretString, // some secret string
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: Same as above – comment probably not needed.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js [hunk 11]
```diff
@@ -58,7 +59,7 @@ passport.deserializeUser((obj, done) => {
 passport.use(new SteamStrategy({
 		returnURL: 'http://localhost:3037/auth/steam/return',
 		realm: 'http://localhost:3037/',
-		apiKey: 'steam api key'
+		apiKey: config.apiKey	// Steam API key
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: Same as above – comment probably not needed.

**Comment by xmwx38** [comment]
@andrewda I have completed all your requests!  Let me know if you have any more.

**Comment by andrewda** [comment]
Thanks so much! Will test this out in a bit

File: Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js (hunk 2 above)
Comment (xmwx38) [reply Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js]: @andrewda Done!

File: Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js (hunk 1 above)
Comment (xmwx38) [reply Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/app.js]: @andrewda Fixed this.

File: Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/config.json (hunk 7 above)
Comment (xmwx38) [reply Chapter 5 - Advanced Web Development/Chapter 5.2 - Authentication/project9/config.json]: @andrewda Added spaces.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js (hunk 3 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js]: @andrewda Fulfilled your request.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js (hunk 4 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js]: @andrewda Removed it.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js (hunk 5 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/app.js]: @andrewda Dealt with.

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/config.json (hunk 6 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.2 - Getting Started/project10/config.json]: @andrewda Added!

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js (hunk 11 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: @andrewda Done!

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js (hunk 10 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: @andrewda Done!

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js (hunk 9 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: @andrewda Done!

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/config.json (hunk 8 above)
Comment (xmwx38) [reply Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/config.json]: @andrewda Done!

Comment (xmwx38) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/config.json]: **Pull request should now be merged with master!**

**Pull request merged**